docs/cloud/recipes/k8s_dask.md

```diff
@@ -0,0 +1,129 @@
+
+# Dask Cluster on Kubernetes
+
+This recipe is for a Flow deployed to Kubernetes which uses a static Dask cluster. The Dask cluster lives on the same Kubernetes cluster that the Flow runs on.
+
+[[toc]]
+
+### Dask YAML
+
+`dask_scheduler.yaml` is the deployment that runs the Dask scheduler.
+
+```yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: dask-scheduler
+  labels:
+    app: dask-scheduler
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: dask-scheduler
+  template:
+    metadata:
+      labels:
+        app: dask-scheduler
+    spec:
+      containers:
+      - name: dask-scheduler
+        image: prefecthq/prefect:latest
+        args:
+          - dask-scheduler
+          - --port
+          - '8786'
+        env:
+          - name: DASK_DISTRIBUTED__SCHEDULER__WORK_STEALING
+            value: "False"
+        ports:
+          - containerPort: 8786
+        resources: {}
+```
+
+`dask_worker.yaml` is the deployment that runs the Dask workers. Notice that setting `replicas: 2` means that there will be two workers in this Dask cluster.
+
+```yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: dask-worker
+  labels:
+    app: dask-worker
+spec:
+  replicas: 2
+  selector:
+    matchLabels:
+      app: dask-worker
+  template:
+    metadata:
+      labels:
+        app: dask-worker
+    spec:
+      containers:
+      - image: prefecthq/prefect:latest
+        args: [dask-worker, dask-scheduler:8786, --no-bokeh, --nthreads, "2", --nprocs, "2"]
```

Comment:
Could we update this to use `nthreads "1"`?  There's a known non-deterministic issue with hanging tasks when using multiple threads that we haven't gotten to the bottom of yet